sentinel/src/lib/infodoc.js

```diff
@@ -1,6 +1,7 @@
 const db = require('../db-pouch'),
   logger = require('../lib/logger'),
-  infoDocId = id => id + '-info';
+  infoDocId = id => id + '-info',
+  getDocId = infoDocId => infoDocId.slice(0, -5);
```

Comment:
I find it really confusing to put functions in the head part which is usually reserved for requirements (similar to imports in Java). What do you think? Is it clearer to have them split out with the rest of the functions?

Reply:
Done!